pkg/scheduler/extender.go

```diff
@@ -70,35 +69,23 @@ func makeTransport(config *schedulerapi.Extender) (http.RoundTripper, error) {
 			cfg.Insecure = true
 		}
 	}
-	tlsConfig, err := restclient.TLSConfigFor(&cfg)
-	if err != nil {
-		return nil, err
-	}
-	if tlsConfig != nil {
-		return utilnet.SetTransportDefaults(&http.Transport{
-			TLSClientConfig: tlsConfig,
-		}), nil
-	}
-	return utilnet.SetTransportDefaults(&http.Transport{}), nil
+	cfg.Timeout = config.HTTPTimeout.Duration
+	return restclient.HTTPClientFor(&cfg)
```

Comment:
this adds the transports Wrappers, and previously it just created a transport from the TLSConfig, no?

Reply:
Only the `DebugWrappers` should be used, which sounds like a good thing?

Reply:
ah, right, we pass an empty rest.Config , that makes sense

/lgtm